apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,57 @@
 import { Component, OnInit } from '@angular/core';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
+import { City } from '../../model/city.model';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { CityStore } from '../../data-access/city.store';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities"
+      [imgUrl]="'assets/img/city.webp'"
+      (newItem)="onNewItem()">
+      <ng-template #rowTemplate let-city>
+        <app-list-item
+          [name]="city.name"
+          [id]="city.id"
+          (itemDelete)="onItemDelete($event)" />
```

Comment:
- You could pass the name in as ng-content here, since it is just being displayed as text.
- You don't need to pass in the id here because you can just us it instead of `$event` in your method call.